pages/favorites.js

```diff
@@ -0,0 +1,36 @@
+import PlantList from "@/components/PlantList";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+
+const BackButton = styled.button`
+  margin-top: 1rem;
+  background-color: #173e14;
+  color: #fff;
+  padding: 0.5rem 1rem;
+  border: none;
+  cursor: pointer;
+  border-radius: 0.5rem;
+`;
+
+const Message = styled.p`
+  color: #fff;
+`;
+
+export default function Favorites({ plants, toggleFavorite }) {
+  const favorites = plants.filter((plant) => plant.isFavorite);
+  const router = useRouter();
+
+  return (
+    <>
+      <h1 style={{ color: "#fff" }}>Favorites</h1>
+
+      {favorites.length > 0 ? (
+        <PlantList plants={favorites} toggleFavorite={toggleFavorite} />
+      ) : (
+        <Message>No plants added to favorites.</Message>
+      )}
+
+      <BackButton onClick={() => router.back()}>Back to Gallery</BackButton>
```

Comment:
Please consider using a link instead of a Button - if you come to this page from anywhere else you will be taken back to that page instead of `/`. If this is what you want you can ignore this suggestion.


```suggestion
      <BackLink href="/">Back to Gallery</BackLink>
```

You'll need to switch the `styled.button` for a `styled(Link)` and `import Link from "next/link"` at the top of the file.